modules/ROOT/pages/configurable-wrapper-properties.adoc

```diff
@@ -0,0 +1,61 @@
+
+* `anypoint.platform.coreservice_base_uri` +
+To mandatory fields since in `on_prem` mode you should run these solutions inside your organization. Default is false |
+|`anypoint.platform.analytics_enabled` | This flag enables or disables analytics statistics. If this property is enabled and `anypoint.platform.analytics_base_uri` is not an URL then `analyticsQueue` and `analyticsProlicyViolationsQueue` are exposed through Mule registry in the properties `_analyticsQueue` and `_analyticsPolicyViolationQueue` | true
+|`anypoint.platform.analytics_push_freq` | Time elapsed between each push to analytics. Time is expressed in seconds.| 5 seconds
+|`anypoint.platform.analytics_cache_capacity` | Cache capacity of the analytics events stored in the internal mapDB used for analytics.| 50000
+|`anypoint.platform.analytics_cache_file` | The name of the analytics report file.| `api-platform-analytics.db`
+|`anypoint.platform.analytics_policy_violations_file` | The name of the analytics policy violations report file.| `api-platform-analytics-policy-violations`
+|`anypoint.platform.analytics_policy_violations_cache_capacity` | Cache capacity of the analytics policy violations events stored in the internal mapDB used for analytics.| 100000
+|`anypoint.platform.analytics_policy_violations_push_freq` | Time elapsed between each push to analytics reporting policy violations. Time is expressed in seconds.| 5 seconds
+|`anypoint.platform.analytics_policy_violations_threshold` | Upper bound to policy violations. If the same policy (policyId) is violated more than the amount of times between each push to analytics, then the message is discarded and logged.| 100000
+|`anypoint.platform.analytics_batch_size` | Upper bound amount of messages to be sent to analytics at once. Each pushes the gateway instance at most this value.| 500000
+|`throttling.persistence_enabled` | This flag enables or disables persistence of the throttling quota.| true
+|`throttling.persistent_data_update_freq` | Time elapsed between each dump of the throttling quota state to disk. Time is expressed in seconds.| 15 seconds
+|`anypoint.platform.x_forwarded_headers_propagation` | Allows generation of X-Forwarded-For and X-Forwarded-Host headers.| true
+|`anypoint.platform.ping_federate_cache_max_size` | Upper bound amount of tokens cached for the ping federate policy.| 100000
+|`anypoint.platform.openid_connect_cache_max_size` | Upper bound amount of tokens cached for the openId policy.| 100000
+|`anypoint.platform.open_am_cache_max_size` | Upper bound amount of tokens cached for the open AM policy.| 100000
+|`anypoint.platform.oauth2_cache_max_size` | Upper bound amount of tokens cached for the OAuth 2 policy. From 3.8.7/3.9.2 onwards (in previous versions, `-oauth2Cache`).| 100000
+|`anypoint.platform.proxy_filter_mule_headers` | When using auto-generated proxies, this option enables or disables the filtering headers that start with `mule_` before sending them to the endpoint behind it.| true
```

Comment:
behind it. -> behind the proxy.   ?

Reply:
@balbifm Does "it" refers to proxy in this sentence? Thank you.

Reply:
endpoint behind it == backend